modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -62,6 +62,10 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 //         health_timeout <duration>
 //         health_status <status>
 //         health_body <regexp>
+//         health_headers {
+//             Host example.com
+//             X-Header-Key value
```

Comment:
We don't tend to use examples here, it should be the syntax definition.

```suggestion
//             <field> <value>
```